src/modules/GameConstants.ts

```diff
@@ -1082,12 +1082,25 @@ export enum StoneType {
     'Metal_alloy',
 }
 
+export enum FossilType {
+    'Helix_fossil',
+    'Dome_fossil',
+    'Old_amber',
+    'Root_fossil',
+    'Claw_fossil',
+    'Armor_fossil',
+    'Skull_fossil',
+    'Cover_fossil',
+    'Plume_fossil',
+    'Jaw_fossil',
+    'Sail_fossil',
+}
+
 export enum FossilPieceType {
-    'None' = -1,
-    'Fossilized Bird',
-    'Fossilized Fish',
-    'Fossilized Drake',
-    'Fossilized Dino',
+    'Fossilized_bird',
+    'Fossilized_fish',
+    'Fossilized_drake',
+    'Fossilized_dino',
 }
```

Comment:
`FossilType` and `FossilPieceType` don't appear to be used anywhere now, is there any reason to keep them?